src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+
 public class SalaryInfo {
+    private final SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        try {
+            Date fromDate = dateFormat.parse(dateFrom);
+            Date toDate = dateFormat.parse(dateTo);
+            HashMap<String, Integer> salaryMap = new HashMap<>();
```

Comment:
In the task requirements you can find 
[Don’t use two-dimension arrays and HashMap(or any other Map)](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#dont-use-two-dimension-arrays-and-hashmapor-any-other-map)